### ISSUE TITLE
crashes when autoScaleMinMaxEnabled == true and no entry in visibleRange

### ISSUE THREAD

**Issue by aelam** [danielgindi/Charts#2376]
It happens when I do like this 
1. `autoScaleMinMaxEnabled = true`  and `axisMinimum = 0`;
```
    _chartView.xAxis.axisMinimum = 0;
    _chartView.autoScaleMinMaxEnabled = true;
```
2. drag/zoom enabled 
3. drag the chart til visibleXRange is 0~29 

I believe that the YAxis gets NaN range because there is no entries for calculating.
Now I have to check the data when chart range changed, and set a good range to it, to keep chart displaying with no data  and with no crash.

If you have any idea  to check or set the YAxis range to make sure it won't get crashed?

**Comment by liuxuan30** [comment]
I recall you had a PR for autoScaleMinMaxEnabled got merged. And this is a new issue? Able to help?

**Comment by aelam** [comment]
No, that one is about calculated a wrong y range ,it's different from this one. This one is about crash
